### DISCUSSION TITLE
Adding Bing Search V7 to Parallel GPT

### DISCUSSION THREAD

**Discussion by shrmn8r** [AntlerVC/firetable#1516] [Support / Q&A]
Hello,

I have successfully added my Open AI key to Parallel GPT and it is working great. The only issue that I would like to fix, is adding web browsing capabilities to this as well. Is it possible to integrate my Bing Search API into this as well? Any help is much appreciated. Thanks

**Comment by Chadha93** [comment]
@shrmn8r, You can try [buildship.com](https://buildship.com/) to address this and give a try and this is compatible with Rowy. Here is the buildship [documentation](https://docs.buildship.com/tutorials/firebase-project). We'll check on adding the Bing Search Integration.
